guides/get-started-microservices-on-kubernetes.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: getstarted
+title: Java Microservices on Kubernetes wih WildFly
+---
+:includedir: ./get-started-microservices-on-kubernetes/_includes
+include::{includedir}/_titles.adoc[]
+:page-liquid:
+
+== Build Java Microservices with WildFly and run them on link:https://kubernetes.io/[Kubernetes, window="_blank"].
+
+=== Intro
+
+In this series, you'll learn HOW-TO to build applications using some of the most common link:https://jakarta.ee/[Jakarta EE, window="_blank"] specifications (e.g. JAX-RS, JPA, JMS, etc.) and run them on link:https://kubernetes.io/[Kubernetes, window="_blank"];
```

Comment:
Also, all of these sentences are finished with `;` instead of `.`. I would say that's not dramatically correct, even more when the sentence finishes a paragraph. Is that intentional?

Reply:
@yersan thank you: I replaced the invalid terms and the ";" with "."